src/test/com/epam/lab/payments/MainRestControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.lab.payments;
```

Comment:
It is incorrect structure. It should be /src/test/java/com/epam/lab/payments

Reply:
moved